src/main/java/seedu/address/model/eatery/Eatery.java

```diff
@@ -103,13 +113,15 @@ public boolean equals(Object other) {
                 && otherEatery.getIsOpen() == (getIsOpen())
                 && otherEatery.getAddress().equals(getAddress())
                 && otherEatery.getCategory().equals(getCategory())
+                && otherEatery.getReviews().equals(getReviews())
```

Comment:
Have you tested this? I'm a bit worried because even if the two eateries are the same, `Eatery#getReviews` calls `Collections#unmodifiableSet` and I'm quite sure it doesn't return identical objects given the same input set 🤔